tox.ini

```diff
@@ -52,10 +52,18 @@ envlist =
     ; Only officially supported Python versions are tested for each Django
     ; major release. Updated list can be found at:
     ; https://docs.djangoproject.com/en/dev/faq/install/#what-python-version-can-i-use-with-django
+    ; The numbers at the end of the environment names
+    ; below mean these dependencies are being used:
+    ; 0: django~=2.0
+    ; 1: django~=3.0
+    ; 2: django>=4.0b1,<5.0 backports.zoneinfo==0.2.1
+    ; 3: django>=4.0b1,<5.0
+    py3{8,9}-test-instrumentation-django-0
```

Comment:
Do you think it would be possible to use the major version of django instead of using an enumeration for the requirements filenames?

Reply:
I think @aabmass pointed this out as well. Seems like we do have a number people who feels that is a better way to indicate different tests. @ocelotl

Reply:
Yeah, we agreed on this approach already.  We can still change it later in another PR. This is only needed for some instrumentarions.

Reply:
I'm reviewing all your PR and am approving the ones that does not have this issue. Since there will be conflicts anyway I think it would be better to update current PRs instead of opening new ones later so you can reuse the current numbering convention and will be easier to review